crates/uv/tests/common/mod.rs

```diff
@@ -312,60 +340,33 @@ impl TestContext {
     /// it can result in tests failing when the index state changes. Therefore,
     /// if you use this, there should be some other kind of mitigation in place.
     /// For example, pinning package versions.
-    pub fn install_without_exclude_newer(&self) -> std::process::Command {
-        let mut command = std::process::Command::new(get_bin());
+    pub fn pip_install_without_exclude_newer(&self) -> Command {
```

Comment:
Honestly I don't think we need a dedicated method for this. People should just `unset_env("EXCLUDE_NEWER")` in the relevant tests?

Reply:
https://github.com/astral-sh/uv/issues/4531